everviz/plugins/_crossplot/crossplot.py

```diff
@@ -0,0 +1,134 @@
+import os
+import pkg_resources
+import dash_html_components as html
+import flask
+
+from uuid import uuid4
+from dash.dependencies import Input, Output
+from webviz_config import WebvizPluginABC
+
+from everviz.plugins._crossplot._layout._crossplot_layout import (
+    _create_graph,
+    _get_radio_dropdown,
+)
+from everviz.plugins._crossplot._callback._crossplot_callback import (
+    _crossplot_update_graph,
+    _get_graph_line,
+)
+from everviz.data.load_csv._get_data import get_data
+from webviz_config.webviz_assets import WEBVIZ_ASSETS
+
+
+class Crossplot(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Crossplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
+        self.dropdown_x_id = f"dropdown-{uuid4()}"
+        self.dropdown_y_id = f"dropdown-{uuid4()}"
+        self.axis_type_x_id = f"radio-{uuid4()}"
+        self.axis_type_y_id = f"radio-{uuid4()}"
+        self.axis_options_x_id = f"radio-{uuid4()}"
+        self.axis_options_y_id = f"radio-{uuid4()}"
+
+        self.data_path = data_path
+        self.set_callbacks(app)
+
+        ASSETS_DIR = pkg_resources.resource_filename(
+            "everviz", os.path.join("assets")
+        )
+
+        WEBVIZ_ASSETS.add(os.path.join(ASSETS_DIR, "axis_customization.css"))
+        # We do this because webwiz currently doesn't give a way to serve
```

Comment:
I agree on the approach, could you also create an issue in everviz so that we don't forget about this comment?

Thanks!